locales/locales.lua

```diff
@@ -120,5 +120,61 @@ Locales = {
         ['autotheft'] = "Tyveri af et køretøj",
         ['officerdown'] = "KOLLEGA SKADET",
         ['persondown'] = "Personen er såret",
+    },
+    
+        ['es'] = {
+        ['unknown_caller'] = "Llamante desconocido",
+        ['caller_unknown'] = "Desconocido",
+        ['caller_local'] = "Local",
+        ['call_from'] = "Llamada de",
+        ['two_door'] = "Dos puertas ",
+        ['three_door'] = "Tres puertas",
+        ['four_door'] = "Cuatro puertas ",
+        ['compact'] = "Compacto", 
+        ['sedan'] = "Sedan",
+        ['suv'] = "Camioneta",
+        ['coupe'] = "Cupé",
+        ['muscle'] = "Auto de alta potencia",
+        ['sports_classic'] = "Deportivo clasico",
+        ['sports'] = "Vehiculo deportivo",
+        ['super'] = "Super Auto",
+        ['motorcycle'] = "Motocicleta",
+        ['offroad'] = "Vehiculo todo-terreno",
+        ['industrial'] = "Vehiculo insdustrial",
+        ['utility'] = "Vehiculo de utilidad",
+        ['van'] = "Van",
+        ['service'] = "Vehiculo de servicio",
+        ['military'] = "Vehiculo militar",
+        ['truck'] = "Camioneta",
+        -- DISPATCH MESSAGES
+        ['vehicleshots'] = "Disparos desde vehículo",
+        ['shooting'] = "Descarga de un arma de fuego",
+        ['melee'] = "Pelea en progreso",
+        ['driveby'] = "Disparos desde vehículos en movimiento",
+        ['speeding'] = "Conducción temeraria",
+        ['autotheft'] = "Robo de un vehículo de motor",
+        ['persondown'] = "La persona esta lesionada",
+        ['storerobbery'] = "Robo de tienda",
+        ['fleecabank'] = "Robo del banco Fleeca",
+        ['paletobank'] = "Robo del Banco Paleto",
+        ['pacificbank'] = "Robo del Banco del Pacífico",
+        ['prisonbreak'] = "Prison Break en progreso",
```

Comment:
This is English, revise to Spansih. Something along the lines of Escape de Carcel